src/MIDebugEngine/AD7.Impl/AD7PendingBreakpoint.cs

```diff
@@ -321,11 +321,6 @@ private int BindWithTimeout()
                 this.SetError(new AD7ErrorBreakpoint(this, ResourceStrings.LongBind, enum_BP_ERROR_TYPE.BPET_SEV_LOW | enum_BP_ERROR_TYPE.BPET_TYPE_WARNING), true);
                 return Constants.S_FALSE;
             }
-            else if (this._BPError != null)
-            {
-                // Ran into some sort of error
-                return Constants.E_FAIL;
```

Comment:
Should the block still exist but return S_FALSE? At least to avoid going into the block starting on line 331?

Reply:
Keeping the deletion but moved away block of 331 to address  #1129